spec/validators/fee/misc_fee_validator_spec.rb

```diff
@@ -22,6 +22,28 @@
 
     describe '#validate_fee_type' do
       it { should_error_if_not_present(fee, :fee_type, 'blank') }
+
```

Comment:
Would it make sense to move the validations for valid miumu quantities from base_fee_validator_spec.rb to here, so that all the validation tests for this fee are in a single location?

Reply:
unfortunately the base_fee_validator is testing alot of the advocate validations (historically this  is because there was only advocate claims in the app). It would be a lot of fiddly work to do a proper job and separate  all the test in a logical manner  - its a significant piece of tech debt beyond the scope of this ticket to rectify IMO